src/core/lookForReallocations.ts

```diff
@@ -62,8 +63,16 @@ export const lookForReallocations = async (
           networkId
         ),
         reallocation: outOfBoundsMarket.aboveRange
-          ? seekForSupplyReallocation(outOfBoundsMarket.id, vault)
-          : seekForWithdrawReallocation(outOfBoundsMarket.id, vault),
+          ? seekForSupplyReallocation(
+              outOfBoundsMarket.id,
+              vault,
+              filterIdleMarkets
```

Comment:
We want to filter out idle markets only from withdraw reallocations (where sometimes we don't want to supply in the idle market)

But withdrawing from idle markets is never a problem
So this should not apply to supply realocations